packages/cli/cli/src/cli-context/TtyAwareLogger.ts

```diff
@@ -11,7 +11,7 @@ export class TtyAwareLogger {
     private tasks: TaskContextImpl[] = [];
     private lastPaint = "";
     private spinner = ora({ spinner: "dots11" });
-    private interval: NodeJS.Timer | undefined;
+    private interval: NodeJS.Timeout | undefined;
```

Comment:
weird -- why did we have to change this?

Reply:
Looks like it's something with my local setup - reverting this continues to pass compilation in CI.